src/arguments.h

```diff
@@ -27,154 +27,203 @@
 extern "C" {
 #endif
 
-#include "dfu-device.h"
 #include "atmel.h"
+#include "dfu-device.h"
 
-#define DEVICE_TYPE_STRING_MAX_LENGTH   6
+#define DEVICE_TYPE_STRING_MAX_LENGTH 6
 /*
  *  atmel_programmer target command
  *
  *  configure {BSB|SBV|SSB|EB|HSB} [--suppress-validation, --quiet, --debug level] value
  *  dump [--quiet, --debug level]
  *  erase [--suppress-validation, --quiet, --debug level]
  *  flash [--suppress-validation, --quiet, --debug level] file
- *  get {bootloader-version|ID1|ID2|BSB|SBV|SSB|EB|manufacturer|family|product-name|product-revision|HSB} [--quiet, --debug level]
- *  launch [--no-reset]
+ *  get {bootloader-version|ID1|ID2|BSB|SBV|SSB|EB|manufacturer|family|product-name|product-revision|HSB} [--quiet,
```

Comment:
This needs a manual line split to avoid splitting in the wrong place, and to get indentation correct

Reply:
I noticed this too. I thought I'd pushed an update to fix.

It actually looks to me like that comment block is vestigial.